tests/cephadm/test_remove_cluster.py

```diff
@@ -0,0 +1,111 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephadmOpsFailure(Exception):
+    pass
+
+
+def _disable_mgr_module(node):
+    """Disable cephadm module"""
+    cmd = "cephadm shell -- ceph mgr module disable cephadm"
+
+    result = node.exec_command(sudo=True, cmd=cmd)
+    if result:
+        log.info(f"Disabled cephadm module sucessfully on node '{node.internal_ip}'")
+        return
+
+    raise CephadmOpsFailure(
+        f"Failed to disable cephadm module on node '{node.internal_ip}'"
+    )
+
+
+def _get_cluster_fsid(installer):
+    """Get cluster FSID"""
+    cmd = "cephadm shell -- ceph fsid"
+    result = installer.exec_command(sudo=True, cmd=cmd)
+
+    if result:
+        log.info("Cluster FSID: {}".format(result[0]))
+        return result[0].strip()
+
+    raise CephadmOpsFailure(
+        f"Failed to get cephadm cluster FSID from installer '{installer.internal_ip}'"
+    )
+
+
+def _remove_cephadm_cluster(nodes, fsid):
+    """Remove cephadm cluster from nodes"""
+    cmd = "cephadm rm-cluster --fsid {} --zap-osds --force".format(fsid)
```

Comment:
Please extend `cephadm` object.

Reply:
Currently no mixin class available for cephadm commands. Will add mixin to support cephadm command for current rm-cluster operation.